examples/nlp_seq/models.py

```diff
@@ -205,14 +194,14 @@ class Transformer(nn.Module):
 
   @nn.compact
   def __call__(self,
-            inputs,
-            train=False,
-            dropout_rate=0.3):
+               *,
+               inputs,
```

Comment:
we usually have `inputs` (or `x`) as the first and only positional argument, i.e.

```
def __call__(self, inputs, *, train):
  ...
```

Reply:
ah. thanks! changed that.